lib/spack/spack/cmd/test.py

```diff
@@ -23,45 +26,67 @@
 
 
 def setup_parser(subparser):
```

Comment:
I really like the options.  Would it be too much or inappropriate to ask for a `spack test clean` option that is equivalent to `spack clean -t`?

Reply:
I don't love the duplication, but I do like the option... worth thinking about.

Reply:
I understand your sentiment about duplication, but the concern is "usability" in the sense that the help for the `test` subcommands can be "complete".  At the same time, you definitely want `spack clean` to be able to clear the directory as well.

It would be nice to have the ability to explicitly clean a named test as well, such as `spack test clean openmpi` while `spack clean -t` would always wipe everything.

Reply:
I'm adding a `spack test remove` command, with an optional `name` argument to remove only an individual test.

Reply:
I like that.  Thanks.

Reply:
Ah, but I can't `spack clean -t` anymore.  Too bad.  It's a **lot** more convenient when you let the tests default to a timestamp.